docs/cli/serve.rst

```diff
@@ -186,6 +186,37 @@ Note that Einhorn will exit if you send it a ``SIGUSR1``. You can instead open u
 
 .. _runtime-metrics:
 
+Prometheus Exporter
+-------------------
+
+When enabled, ``baseplate-serve`` will start up a Prometheus exporter on HTTP
+port 6060. The exporter is designed for multiprocess use, like if running under
+Einhorn.  Each worker process writes its metrics to a file on disk and runs its
+own copy of the exporter, all listening on the same port. Any given worker can
+serve the metrics for all workers from the same pod. It doesn't matter which
+one is hit by the Prometheus collector.
+
+To enable the exporter, install the ``prometheus-client`` package from PyPI.
+
+.. code-block:: console
+
+   $ pip install prometheus-client
+
+The ``PROMETHEUS_MULTIPROC_DIR`` environment variable must be set to the path
+to an extant writeable directory. This is handled automatically in the official
+base Docker images.
```

Comment:
Is there a PR to add this env var  `PROMETHEUS_MULTIPROC_DIR` to the official base Docker image?

Reply:
yup! it's at reddit/docker-baseplate-py#57 on our github enterprise